packages/fx-core/src/common/samples-config.json

```diff
@@ -50,6 +50,20 @@
             "shortDescription": "Search and Share an NPM Package in Teams and Outlook (Web)",
             "fullDescription": "NPM Search Connector is a Messaging Extension that runs across M365 including Teams and Outlook (Web). It allows you to perform a quick search for a NPM package and insert package details into conversations for easy sharing. The front end is built with Adaptive Cards and the backend is an Azure Bot Service.",
             "tags": ["Message Extension", "JS", "Outlook"]
+        },
+        {
+            "id": "hello-world-tab",
+            "title": "Hello World Tab",
+            "shortDescription": "Hello World app of Microsoft Teams Tab app",
+            "fullDescription": "This is a Hello World app of Microsoft Teams Tab app which accomplishes very simple function like single-sign on. You can run this app locally or deploy it to Microsoft Azure. This app only has a Tab frontend.",
+            "tags": ["Tab", "TS", "Ready to F5 Preview"]
+        },
+        {
+            "id": "hello-world-tab-with-backend",
+            "title": "Hello World Tab with Backend",
+            "shortDescription": "Hello World app of Microsoft Teams Tab app",
```

Comment:
Is it expected to have the same short description for the two samples?

Reply:
Yes. Is there limitation for length of short description?